classes/session/file.php

```diff
@@ -59,7 +59,11 @@ public function gc()
 						strpos($file, $this->config['cookie_name'].'_') === 0 and
 						filemtime($this->config['path'] . $file) < $expire)
 					{
-						@unlink($this->config['path'] . $file);
+						clearstatcache(true, $this->config['path'] . $file);
+						if (is_file($this->config['path'] . $file))
+						{
+							@unlink($this->config['path'] . $file);
```

Comment:
And this "@" can be removed, as the file should exist after a `clearstatcache()` and an `is_file()` check